### ISSUE TITLE
ascii decoding issues

### ISSUE THREAD

**Issue by Ekultek** [Ekultek/Dagon#37]
```
baal@baal-Aspire-5733Z:~/bin/python/dagon$ python dagon.py -c 0cb6948805f797bf2a82807973b89537 --bruteforce -w /usr/share/dict/american-english -A 1000
Dagon .. Advanced Hash Manipulation v1.12.26.45(dev)
Clone: https://github.com/ekultek/dagon.git


[*] Starting up at 12:22:55..

[12:22:55 INFO] Reading from, /usr/share/dict/american-english..
[12:22:55 INFO] Using algorithm, NTLM..
[12:22:55 CRITICAL] dagon.py failed with error code: ''ascii' codec can't decode byte 0xc3 in position 6: ordinal not in range(128)'. Creating a wordlist..
[12:22:55 WARNING] It is highly advised to use a dictionary attack over bruteforce. Bruteforce requires extreme amounts of memory to accomplish and it is possible that it could take a lifetime to successfully crack your hash. To run a dictionary attack all you need to do is pass the wordlist switch ('--wordlist PATH') with the path to your wordlist. (IE: --bruteforce --wordlist ~/dicts/dict.txt)
^C[12:22:56 CRITICAL] User exited process...

```

**Comment by Ekultek** [comment]
Fixed during next push:

```
baal@baal-Aspire-5733Z:~/bin/python/dagon$ python dagon.py -c 0cb6948805f797bf2a82807973b89537 --bruteforce -w /usr/share/dict/american-english -A 1000
Dagon .. Advanced Hash Manipulation v1.12.27.46(dev)
Clone: https://github.com/ekultek/dagon.git


[*] Starting up at 12:37:37..

[12:37:37 INFO] Reading from, /usr/share/dict/american-english..
[12:37:37 INFO] Using algorithm, NTLM..
---------------------------------------------------------------------------
[+] Match found:
---------------------------------------------------------------------------
[+] Clear Text: test
[+] Hash: 0cb6948805f797bf2a82807973b89537
[+] Tries attempted: 90190
[+] Algorithm Used: NTLM
---------------------------------------------------------------------------

[*] Shutting down at 12:37:37..

baal@baal-Aspire-5733Z:~/bin/python/dagon$ 

```

**Comment by Ekultek** [comment]
fixed via 15d4ee5
